tests/publiccloud/run_ltp.pm

```diff
@@ -145,6 +145,10 @@ sub run {
     my $runltp_ng_branch = get_var("LTP_RUN_NG_BRANCH", "master");
     assert_script_run("git clone -q --single-branch -b $runltp_ng_branch --depth 1 $runltp_ng_repo");
     $instance->run_ssh_command(cmd => 'sudo CREATE_ENTRIES=1 ' . get_ltproot() . '/IDcheck.sh', timeout => 300);
+    if(is_ec2_arm()){
+        record_info('Workaround', 'AWS Nitro virtualization not recognized by systemd-detect-virt see bsc#1190440');
+        $instance->run_ssh_command(cmd => 'sudo echo "LTP_VIRT_OVERRIDE=amazon" >> /etc/environment');
```

Comment:
this doesn't work if you are a user... I think it should be
```suggestion
        $instance->run_ssh_command('echo "LTP_VIRT_OVERRIDE=amazon" | sudo tee -a /etc/environment');
```

Reply:
no problem to change it but just theoretical question - what the benefit of `tee -a` in comparison with `>>` .

Reply:
that it will work with sudo.
In your case, the `echo` command is executed as `root` but the redirect get the `USER` permissions. And a normal `user` do not have write access to `/etc`.
Just try your command on your computer.